extensions/wikia/WikiFactory/Close/verifyWIthEmptyDB.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+// make Wikia\Logger\Loggable trait available at a run-time
+require_once( __DIR__ . '/../../../../lib/composer/autoload.php' );
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+require_once __DIR__ . '/gcs_bucket_remover.php';
+
+class VerifyWithEmptyDB extends Maintenance {
+
+	use Wikia\Logger\Loggable;
+
+	public function __construct() {
+		parent::__construct();
+		$this->addOption( 'first', 'Run only once for first wiki in queue' );
+		$this->addOption( 'from', 'Starting offset for wiki to be closed' );
+		$this->addOption( 'dry-run', 'List wikis that will be removed and quit' );
+		$this->addOption( 'limit', 'Limit how many wikis will be processed', false, true );
+		$this->addOption( 'sleep', 'How long to wait before processing the next wiki', false, true );
+		$this->addOption( 'cluster', 'Run for a given cluster only', false, true );
+	}
+
+	/**
+	 * 1. go through all wikis which are marked for closing and check which one
```

Comment:
I think this comment should be moved up, as it nicely explains what this maintenance class does

Reply:
Unfortunately I think that this comment is not about this maintenance script, it's probably copy/pasta from Close/maintennance scirpt

Reply:
Oh, that's even worse :) Then we should remove it and add a proper comment explaining the purpose and usage of this script at the top of this file.